assets/scss/utility/_display.scss

```diff
@@ -0,0 +1,19 @@
+$screenSizes: (
```

Comment:
This is overcomplicating a problem, just a simple utility class is required; we're not rewriting tailwind or bootstrap.